lib/defaultConfig.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  drupal: {
+    path: "./.breakpoints.config.yml",
+    themeName: "themeName",
+  },
+  js: {
+    enabled: true,
+    path: "./breakpoints.js",
+    es6: true,
```

Comment:
Maybe switch from a boolean type to an enum? I think Node.js’ [type](https://nodejs.org/api/packages.html#type) is a good “standard”:

```yaml
type: module | commonjs
```